api/Controllers/AccountsController.cs

```diff
@@ -36,10 +34,20 @@ public async Task<IActionResult> CreateAccount([FromBody] Login request)
             };
             _hotelsDb.Add(user);
             await _hotelsDb.SaveChangesAsync();
-            return NoContent();
+            return CreatedAtRoute(
+                routeName: "GetLink",
+                routeValues: new { id = user.Id },
+                value: user);
         }
 
+        [Route("{id}")]
+        [HttpGet("{id}", Name = "GetLink")]
+        [Authorize]
+        public async Task<Account> GetAccount([FromRoute] int Id) =>
+            await _hotelsDb.Accounts.SingleOrDefaultAsync(account => account.Id == Id);
```

Comment:
Is it a good practice to use database entities as a return value in controller actions?